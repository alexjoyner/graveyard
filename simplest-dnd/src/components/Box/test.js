import React from 'react';
import {shallow} from 'enzyme';
import {Box} from '.';

describe('Box', () => {
    it('renders properly', () => {
        let component = shallow(<Box/>);
        expect(component).toMatchSnapshot();
    })
})