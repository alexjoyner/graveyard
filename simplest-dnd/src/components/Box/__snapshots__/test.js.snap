// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Box
    connectDragSource={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): undefined,
  Symbol(enzyme.__nodes__): Array [
    undefined,
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
    "disableLifecycleMethods": true,
  },
}
`;
